bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/SitemapSubscriptionService.java

```diff
@@ -225,4 +249,28 @@ private Sitemap getSitemap(String sitemapName) {
         return null;
     }
 
+    @Override
+    public void modelChanged(String modelName, EventType type) {
+
```

Comment:
drop this empty line - good for separating logic blocks, but doesn't add any value here.